.github/workflows/unit-tests.yaml

```diff
@@ -0,0 +1,55 @@
+name: Unit test
+
+on:
+  release:
+    types: [created]
+  pull_request:
+    branches:
+      - main
```

Comment:
we still use master branch, so I guess we need to change that here

Reply:
ah yes.